### ISSUE TITLE
Share output from npm during install

### ISSUE THREAD

**Issue by jakirkham** [QuantStack/ipytree#3]
Would be nice to see the output from `npm` during the `pip install` both to see what it is doing and aid in diagnosing any build or installation issues.

**Comment by martinRenou** [comment]
I totally agree with you. I used the [widget-cookiecutter](https://github.com/jupyter-widgets/widget-cookiecutter) for creating this project, do you mind opening the same issue on this repo instead? I would be happy to port the changes here as well once it is fixed upstream :)

**Comment by jakirkham** [comment]
Sure, have added issue ( https://github.com/jupyter-widgets/widget-cookiecutter/issues/46 ). Please feel free to fill in anything I may have missed.

**Comment by martinRenou** [comment]
Thanks :)
